ts/features/itwallet/common/saga/index.ts

```diff
@@ -18,6 +19,9 @@ export function* watchItwSaga(): SagaIterator {
   // Check if the fiscal code is enabled, to enable the L3
   yield* fork(checkFiscalCodeEnabledSaga);
 
+  // Check if the device has the NFC Feature
+  yield* call(checkHasNfcFeatureSaga);
```

Comment:
Since it's async, we can use `fork`. What do you think?
```suggestion
  yield* fork(checkHasNfcFeatureSaga);
```

Reply:
Applied in https://github.com/pagopa/io-app/pull/6985/commits/852ed0f5c3acbca91da8fe59ec6f5d8b6c80de85